input/fsh/MDI/instances/Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-11.fsh

```diff
@@ -1,11 +1,6 @@
 Instance: Inline-Instance-for-bundle-mdi-to-edrs-a-freeman-11
 InstanceOf: USCorePractitionerProfile
```

Comment:
should this be InstanceOf: PractitionerVitalRecordsNew

Reply:
Got rid of all of the inline instances....didn't rename example instances.